### PR TITLE
pkp/pkp-lib#7874 Unsubscribe link for notification emails

### PR DESCRIPTION
**Pull request by Vitaliy-1**

### CODE REVIEW

File: classes/controllers/grid/issues/IssueGridHandler.php [hunk 1]
```diff
@@ -598,22 +599,50 @@ public function publishIssue($args, $request)
...
 
+            [$userIdsToNotify, $userIdsToNotifyAndMail] = [
+                $userIdsToNotify->diff($userIdsToMail),
+                $userIdsToNotify->intersect($userIdsToMail)
+            ];
+
             $jobs = [];
-            foreach ($userIdsToMail->chunk(Mailer::BULK_EMAIL_SIZE_LIMIT) as $mailUserIds) {
+            foreach ($userIdsToNotify->chunk(PKPNotification::NOTIFICATION_CHUNK_SIZE_LIMIT) as $notifyUserIds) {
+                $notifyJob = new IssuePublishedMailUsers(
```
Comment (NateWr) [inline classes/controllers/grid/issues/IssueGridHandler.php:629]: Is this the right job? It looks the same as the mail job.

File: registry/emailTemplates.xml [hunk 2]
```diff
@@ -70,4 +69,7 @@
 	<email key="ANNOUNCEMENT" subject="emails.announcement.subject" body="emails.announcement.body"/>
 	<email key="EDITORIAL_REMINDER" can_disable="0" from_role_id="16" to_role_id="1048576" subject="emails.editorialReminder.subject" description="emails.announcement.description" body="emails.editorialReminder.body"/>
 	<email key="DISCUSSION_NOTIFICATION" subject="discussion.notification.subject" body="discussion.notification.body"/>
+	<email key="VERSION_CREATED" subject="emails.versionCreated.subject" body="emails.versionCreated.body"/>
+	<email key="REVIEW_COMPLETE" subject="emails.reviewComplete.subject" body="emails.reviewComplete.body"/>
+	<email key="REVIEW_EDITED" subject="emails.reviewEdited.subject" body="emails.reviewEdited.body"/>
```
Comment (NateWr) [inline registry/emailTemplates.xml:74]: Call this `REVIEW_EDIT` so it has the same verb tense as the other review emails. You can move these two up with the other `REVIEW_*` emails too, just to keep them together.

File: classes/controllers/grid/issues/IssueGridHandler.php (hunk 1 above)
Comment (Vitaliy-1) [reply classes/controllers/grid/issues/IssueGridHandler.php:629]: Yes, it's should be one job but better to rename

File: Jobs/Notifications/IssuePublishedNotifyUsers.php [hunk 3]
```diff
@@ -17,44 +17,82 @@
...
+    // Whether to send notification email
+    protected bool $sendEmail = false;
+
+    public function __construct(
+        Collection $recipientIds,
+        int $contextId,
+        Issue $issue,
+        string $locale,
+        ?User $sender = null,
+        bool $sendEmail = false
```
Comment (NateWr) [inline Jobs/Notifications/IssuePublishedNotifyUsers.php:54]: Maybe too late to bother with, but could these last two arguments be reduced to one? 

```
@param ?User $sender The sender of the email. Leave null to not send an email.
```

File: locale/en_US/manager.po [hunk 4]
```diff
@@ -2140,3 +2140,11 @@ msgstr "Archive Request"
 msgid "mailable.lockssNewArchive.name"
 msgstr "Archive Request (New)"
 
+msgid "mailable.publicationVersionNotify.name"
+msgstr "Publication Version Notification"
+
+msgid "mailable.publicationVersionNotify.description"
+msgstr ""
+"This email automatically notifies assigned editors about new version of the "
```
Comment (NateWr) [inline locale/en_US/manager.po:2148]: > This email automatically notifies assigned editors when a new version of the submission is created.

File: locale/en_US/manager.po [hunk 5]
```diff
@@ -2140,3 +2140,11 @@ msgstr "Archive Request"
 msgid "mailable.lockssNewArchive.name"
 msgstr "Archive Request (New)"
 
+msgid "mailable.publicationVersionNotify.name"
```
Comment (NateWr) [inline locale/en_US/manager.po:2143]: Should this key (and the one below) be in pkp-lib?

File: locale/en_US/manager.po [hunk 6]
```diff
@@ -2140,3 +2140,11 @@ msgstr "Archive Request"
...
+msgstr "Publication Version Notification"
```
Comment (NateWr) [inline locale/en_US/manager.po:2144]: > New Version Created

**Pull request merged**